src/sentry/utils/samples.py

```diff
@@ -134,18 +135,26 @@ def load_data(
         language = platform_data["language"]
 
     samples_root = os.path.join(DATA_ROOT, "samples")
-    all_samples = {f for f in os.listdir(samples_root) if f.endswith(".json")}
 
     for platform in (platform, language, default):
         if not platform:
-            continue
+            raise ValueError("platform cannot be an empty value")
```

Comment:
This one I'm not too sure on. If an empty string or `None` was provided we would continue on silently. It made more sense to me to raise an explicit exception to better detect issues/bugs.